helm/cluster-openstack/values.schema.json

```diff
@@ -0,0 +1,192 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "properties": {
+        "baseDomain": {
+            "type": "string"
+        },
+        "bastion": {
+            "type": "object",
+            "properties": {
+                "bootFromVolume": {
+                    "type": "boolean"
+                },
+                "diskSize": {
+                    "minimum": 1,
+                    "type": "integer"
+                },
+                "flavor": {
+                    "type": "string"
+                },
+                "image": {
+                    "type": "string"
+                }
+            },
+            "required": [
+                "flavor",
+                "image"
```

Comment:
Isn't `diskSize` required?

Reply:
AFAIK `diskSize` only applies if `bootFromVolume` is true (i.e. using root volumes). And we can't do dependent validation with JSON schema. @ericgraf Can you confirm that you can't set the disk size using ephemeral volumes?

Reply:
Looks like you're right. Ephemeral root volume is defined by the flavour or image.

Reply:
We could add some more sophisticated validation like here for that https://gigantic.slack.com/archives/C01BYMF6RN0/p1648218775082879

Reply:
I feel like that kind of validation is separate from adding a values schema so I'll merge this PR without that.